trussworks/react-uswds
# feat: add combo box component

# Add Combo Box

This PR is a first pass at adding the [combo box](https://designsystem.digital.gov/components/form-controls/#combo-box) component. We are merging to a feature branch for now since we anticipate refactoring the component.

The initial implement uses a prop for handling the options list and `setFieldValue` to control the input value.  

There are significant tests added, which will help be a guard us as future refactors are made. 

## Related Issues or PRs

related to #170 

## How To Test

1. Open storybook and verify that the behavior matches what is shown in the [USWDS version](https://designsystem.digital.gov/components/form-controls/#combo-box).
